.conda-build/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: psuade-lite
-  version: '1.9.0'
```

Comment:
@kbuma can we revert this change, so that the version is 1.9.0? IIRC `conda-build` supports multiple builds with the same version tag, which seems the appropriate thing to do here since there are no actual changes in the codebase.

Reply:
sure

Reply:
@lbianchi-lbl done